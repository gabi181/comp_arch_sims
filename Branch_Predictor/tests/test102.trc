16 7 19 3 global_history local_tables not_using_share
0x39D6009C N 0x3E45A614
0x75B68D8C N 0x122A5ED4
