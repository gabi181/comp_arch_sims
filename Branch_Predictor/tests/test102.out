0x39d6009c N 0x39d600a0
0x75b68d8c N 0x75b68d90
flush_num: 0, br_num: 2, size: 4903b
